package.json

```diff
@@ -55,18 +51,14 @@
   "devDependencies": {
     "ajv-keywords": "^3.4.0",
     "chai": "^4.2.0",
-    "chai-like": "^1.1.1",
-    "chai-sinon": "^2.8.1",
-    "coveralls": "^3.0.2",
+    "coveralls": "^3.0.3",
```

Comment:
Please run `depcheck` make sure we don't have missing dependencies or redundant ones